cmake/CompilerWarnings.cmake

```diff
@@ -71,7 +71,7 @@ function(set_file_warnings)
     
     # For now if we're using MSVC-like clang interface on Windows
     # we'll disable warnings as errors 
```

Comment:
I think this comment needs to be updated because we disable warnings are errors with clang too (not only MSVC-like clang).